include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -533,7 +552,7 @@ struct __usm_host_or_buffer_storage
                      : __usm_host_or_buffer_accessor<_T>(__cgh, __sycl_buf.get());
     }
 
-    auto
+    _T
     __get_value(size_t idx = 0)
```

Comment:
Can we declare this function as `const` ?

Reply:
Done.